sycl/include/sycl/ext/oneapi/matrix/matrix-unified-utils.hpp

```diff
@@ -11,6 +11,7 @@
 #include <optional>                     // std::optional
 #include <string_view>                  // std::string_view
 #include <sycl/__spirv/spirv_types.hpp> // __spv namespace
+#include <sycl/ext/oneapi/bfloat16.hpp> // bfloat16
```

Comment:
Where was "bfloat16.hpp" coming from before the explicit include you are adding?

Reply:
I assume it was included to the headers where matrix-unified-utils.hpp was included.

Reply:
No, it is not  in https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/__spirv/spirv_types.hpp